Tools/MonoGame.Tools.Tests/ContentCompilerTest.cs

```diff
@@ -17,9 +17,6 @@
 
 namespace MonoGame.Tests.ContentPipeline
 {
-#if MACOS
-    [Ignore("Hanging on Mac in CI?")]
-#endif
```

Comment:
Dunno if it's related, but there are some tests which won't run on threads and will just block. As far as I have looked into this, there is not system to ensure that tests are running on the main thread with the current test runner (NUnit).